mri/reconstructors/self_calibrating.py

```diff
@@ -205,6 +205,7 @@ def reconstruct(self, kspace_data, optimization_alg='pogm', x_init=None,
         if recompute_smaps and \
                 not check_if_fourier_op_uses_sense(self.fourier_op):
             # Extract Sensitivity maps and initialize gradient
+            print('start Smaps extraction')
```

Comment:
Let us not have print statements without any verbosity, either remove this or add it based on Verbosity.

Reply:
Ok I'll remove it, it was for my personal understanding